siliconcompiler/tools/vpr/_json_constraint.py

```diff
@@ -87,11 +88,11 @@ def map_constraints(json_generic_constraints,
                     constraints_map[design_pin_assignment]['subtile'])
 
             else:
-                design_pin_constraint_assignment = (0, 0, 0)
+                errors += 1
```

Comment:
maybe you can add a call to `chip.logger.error('')` with a message about which pin has issues

Reply:
Done.